src/components/baseComponent.ts

```diff
@@ -78,6 +78,14 @@ export abstract class MgtBaseComponent extends LitElement {
 
   private static _useShadowRoot: boolean = true;
 
+  /**
+   * Get the version value for this component.
+   *
+   * @returns {string}
+   * @memberof MgtBaseComponent
+   */
+  public version: string = '1.0.0';
```

Comment:
The version should be the same as the package version, no need to have separate versions

Reply:
So we don't have any interest in versioning controls individually? We aren't guaranteed to need to rev the version for controls we don't touch between releases.

Reply:
We spoke directly. I'll remove this.